### DISCUSSION TITLE
[灌水]刚刚缴获了日军物资

### DISCUSSION THREAD

**Discussion by EhuanyingK** [share121/inter-knot#1358] [其他]
据日军所说是“超！罕见的表情包”（不懂就问日军物资这么稀有的吗
![7841791e4ef57d7695d12103234f6f7edb35ba9d_raw](https://github.com/user-attachments/assets/7f6155db-0688-4e6e-befb-0085431a861e)

**Comment by lllliufeng** [comment]
没错，这就是罕见
啊不对，就是稀有

**Comment by shenguojia123** [comment]
好稀有的物资

**Comment by Mayf1ower** [comment]
骂谁罕见！骂谁罕见！

**Comment by xyz114** [comment]
科普一下，这是一种莲类植物，生长在雪山的东部，极为罕见，故名东雪莲
